playground/src/preview/preview.js

```diff
@@ -27,10 +28,10 @@ const PARAM_HEIGHT = 'height';
 const MOBILE_BREAK_POINT = 767;
 
 export const EVENT_AMP_BIND_READY = 'event-amp-bind-ready';
-
 export const EVENT_AMP_BIND_NEW_STATE = 'event-amp-bind-new-state';
 
 export function createPreview(container) {
+  if (!container) return;
```

Comment:
Same as with the others. It's a bit weird to have a method called create which doesn't create something.